contracts/src/GatewayDiamond.sol

```diff
@@ -26,13 +26,13 @@ contract GatewayDiamond {
     GatewayActorStorage internal s;
 
     struct ConstructorParams {
-        SubnetID networkName;
-        uint256 bottomUpCheckPeriod;
         // deprecated (for now): no `msgFee` currenlty charged for cross-net messages
         uint256 msgFee;
+        uint256 bottomUpCheckPeriod;
+        uint16 activeValidatorsLimit;
         uint8 majorityPercentage;
+        SubnetID networkName;
```

Comment:
Maybe the day will come when we can rename this variable to `subnetId` :crossed_fingers:

Reply:
Is there any practical reason for reordering these?

Reply:
Just to save more space in memory and in the storage.